packit/config/job_config.py

```diff
@@ -145,7 +145,7 @@ def identifier(self):
         original_identifier = super().__getattr__("identifier")
         if original_identifier is None:
             return self._view_for_package
-        return original_identifier
+        return original_identifier + ":" + self._view_for_package
```

Comment:
Can you please rename the `_view_for_package` parameter to make it more explicit what it contains?

(Maybe `view_identifier`/`package_view_identifier`/....)

Reply:
The _view_for_package is not related with the identifier, is the name of the package for which the `JobConfigView` has been created. But since it is a string I use it to customize the identifier...